docs/content/1.getting-started/0.index.md

```diff
@@ -1,30 +1,42 @@
-# Getting Started
+# 🚀 Getting Started with Yassi
 
-## Installation
+Welcome to Yassi, the HomeAssistant integration for your Samsung Soundbar. This guide will help you get up and running in no time.
 
-### HACS (official)
+## 📦 Installation Options
 
-> ⚠️ Not done yet, hopefully soon.
+### HACS (Home Assistant Community Store)
 
+#### 🌟 Official Repository (Coming Soon)
+The Yassi integration will be available through the official HACS repository shortly. Stay tuned for updates.
 
-### HACS (custom repository)
+#### ➕ Custom Repository
+In the meantime, you can manually add this repository to HACS:
+1. Navigate to HACS in your HomeAssistant.
+2. Go to 'Settings' and add this repository URL as a custom repository.
+3. Search for 'Yassi' under integrations and proceed with the installation.
```

Comment:
```suggestion
1. Click the following button and 'open link':
[![Open your Home Assistant instance and open a repository inside the Home Assistant Community Store.](https://my.home-assistant.io/badges/hacs_repository.svg)](https://my.home-assistant.io/redirect/hacs_repository/?owner=samuelspagl&repository=ha_samsung_soundbar&category=integration)
2. Click 'add' to add the custom repository.
3. Download 'Yassi' and reboot Home Assistant.
```

Reply:
Didn't know that a redirect like that is possible, cool. Thanks!